src/components/NewsRoomList.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import FilterList from "./FilterList";
+import NewsRoomCard from "./NewsRoomCard";
+import { newsItems } from "../files/NewsItems";
+import ListCounter from "./ListCounter";
+import useNewsRoom from "../hooks/useNewsRoom";
+import { Alert, Button } from "@baltimorecounty/dotgov-components";
+
+const NewsRoomList = () => {
+  const { hasError, newsRoomItems = [], isLoading } = useNewsRoom();
+  const [filteredItems, setFilteredItems] = useState([]);
```

Comment:
not blocking, but the filtering logic doesn't really need to be here. This will come in the next user story. Since we know for sure this is coming, i don't think its worth removing, but generally we want these prs as small as possible.

Reply:
changed